tools/cldr-apps/js/src/views/MainHeader.vue

```diff
@@ -1,7 +1,17 @@
 <template>
   <header id="st-header">
+    <a-spin v-if="!loaded" :delay="250" />
     <ul>
-      <li>{{ stVersionPhase }}</li>
+      <li>
+        {{ stVersion }} {{ stPhase }}
+        <span
+          class="ddlException"
+          v-if="ddlException"
+          title="Note: As a non-TC DDL locale, this phase has been extended."
```

Comment:
ideally this title might include ddlPhase, since "extended" is vague -- extended to what?